### ISSUE TITLE
Choice of License 

### ISSUE THREAD

**Issue by miyer26** [UBC-MDS/magmaviz#47]
Choice of License: MIT 

Reasons: 
Our package is a general purpose visualization package that is designed to be used commercially by a wide range of audiences. Therefore the MIT license, which is simple, familiar and non-restrictive, is ideal for this project. Furthermore, our project is open-sourced - all the code is available in this repository. This license provides commercial use permission, distribution, modification and private use which allows anyone who wants to use the package and view the source code to do so. As the MIT license is commonly used, people will not be dissuaded from using the package. The only drawback with the use of this license is that there is limited liability, however as this is a simple project, we believe this should not be an issue. Furthermore, this package does not have any reports or other analytical components hence a Creative Commons license is unnecessary. 

The same license is also used for the [R package](https://github.com/UBC-MDS/magmavizR).

**Comment by iamMoid** [comment]
Nicely drafted @miyer26. Thank you!

**Comment by ruben1dlg** [comment]
I agree with your comment @miyer26 , I think our project could be useful to  a wide range of users and there is no need to limit it. I agree that having it open-source is also the best, so that users can check what the package is actually doing and understand better what the package is doing when calling one of its functions.

**Comment by shyan0903** [comment]
In addition to what @miyer26 and @ruben1dlg have mentioned above, I would like to emphasize another great advantage of using the MIT license. It is its license compatibility, which means that it allows for pieces of software with different licenses to be distributed together. We believe this feature will greatly support our audience to freely create, combine, and publish new software programs for various purposes.

**Comment by miyer26** [comment]
It was great discussing the choice of license with you all!
